e2e/install_test.go

```diff
@@ -716,6 +703,19 @@ func TestResetAndReinstallAirgap(t *testing.T) {
 		t.Fatalf("fail to install embedded-cluster on node %s: %v", tc.Nodes[0], err)
 	}
 
+	if err := setupPlaywright(t, tc); err != nil {
+		t.Fatalf("fail to setup playwright: %v", err)
+	}
+	if _, _, err := runPlaywrightTest(t, tc, "deploy-app"); err != nil {
+		t.Fatalf("fail to run playwright test deploy-app: %v", err)
+	}
```

Comment:
There is a function called `setupPlaywrightAndRunTest` that executes both things at the same time.